usersync/usersyncers/syncer_test.go

```diff
@@ -60,7 +60,7 @@ func TestNewSyncerMap(t *testing.T) {
 	}
 
 	adaptersWithoutSyncers := map[openrtb_ext.BidderName]bool{
-		openrtb_ext.BidderTappx: true,
+		openrtb_ext.BidderTappx:   true,
```

Comment:
An extra space got in here somehow ... otherwise looks good.

Reply:
Those lines were aligned by `gofmt -w usersync/usersyncers/syncer_test.go`

But when I remove spaces, **validate.sh** doesn't pass:
>  ./validate.sh --nofmt --cov --race 10
gofmt needs to be run, 1 files have issues.  Below is a list of files to review:
usersync/usersyncers/syncer_test.go

So Travis CI would not pass.

Reply:
Odd, ow well, not worth wasting too much time over.